app/controllers/picks_controller.rb

```diff
@@ -19,7 +19,15 @@ def show
   end
 
   def create
-    @pick = Pick.new(pick_params)
+    @pick = Pick.create(pick_params)
+    respond_to do |format|
+      format.jsonapi { render jsonapi: @pick }
+    end
+  end
+
+  def destroy
+    @pick = Pick.find(params[:id])
+    @pick.destroy
```

Comment:
That approach throws
`undefined method `destroy' for nil:NilClass`

So I'm keeping it the same